### PR TITLE
Implemented add book feature

### PR DESCRIPTION
**Pull request by SergiuPopistasItSchool**
Implementation of ticket Number 1.
Add book feature implementation

### CODE REVIEW

File: ItSchoolBookApp.py [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+    #imporint csv lib
+    import csv
+    with open('booksDB.csv', 'w') as file:
+        writer = csv.DictWriter(file,fieldnames=[
+            "BookName", "AuthorName", "SharedWith", "IsRead"
+        ])
+        writer.writerow({"BookName": book_name,
+                         "AuthorName": author_name})
+    print("Book was added successfully")
+
```
Comment (Bahrim00) [inline ItSchoolBookApp.py:13]: Extra space. Noot needed

Comment (SergiuPopistasItSchool) [inline ItSchoolBookApp.py:13]: Extra space. Not needed

File: ItSchoolBookApp.py [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+
```
Comment (Bahrim00) [inline ItSchoolBookApp.py:14]: Extra space

File: ItSchoolBookApp.py (hunk 1 above)
Comment (SergiuPopistasItSchool) [reply ItSchoolBookApp.py:13]: Needed by python

**Review by Bahrim00 — APPROVED** [review]

**Pull request merged**